descwl_coadd_task/tests/test_warps.py

```diff
@@ -0,0 +1,362 @@
+from __future__ import annotations
+
+import unittest
+
+import lsst.afw.cameraGeom.testUtils
+import lsst.afw.image as afw_image
+import lsst.skymap as skyMap
+import lsst.utils.tests
+import numpy as np
+from lsst.afw.detection import GaussianPsf
+from lsst.daf.butler import DataCoordinate, DimensionUniverse
+from lsst.pipe.base import InMemoryDatasetHandle
+from lsst.pipe.tasks.coaddBase import makeSkyInfo
+
+from descwl_coadd_task import MakeShearWarpConfig, MakeShearWarpTask
+
+PIXEL_SCALE = 0.2
+"""Pixel scale in arcseconds"""
+
+
+class MakeWarpTestCase(lsst.utils.tests.TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        # Docstring inherited.
+        # This method contains the setup that needs to be done only once for
+        # all the tests.
+        cls.ny = 4000
+        cls.nx = 4072
```

Comment:
These values are a bit high for unit tests, but mocking up an appropriate geometry that is sufficiently non-trivial was taking up time and I decided to base this off of a DC2 image.

Reply:
In my commits (since discarded) I was able to use smaller images. Should I re-push those changes?

Reply:
This has to be adjusted together with a few other changes in `_make_data`. Is it a question of simulation time to have larger images?

Reply:
Yes time/space etc.  I'll worry about that in other PRs